alchemy.lic

```diff
@@ -5,11 +5,6 @@
 custom_require.call(%w[common common-crafting common-items common-travel common-arcana])
```

Comment:
Can `common-arcana` be removed? I don't see where it is used

Reply:
Thank you!